package.json

```
@@ -71,5 +71,5 @@
   "lint-staged": {
     "*.md": "yarn run ci:fix"
   },
-  "packageManager": "yarn@3.5.0"
+  "packageManager": "yarn@3.6.0"
```

Please keep the changes scoped to the tutorial. Feel free to open a subsequent PR for those changes.

```suggestion
  "packageManager": "yarn@3.5.0"
```